src/TestFramework/Coverage/JUnit/JUnitTestExecutionInfoAdder.php

```diff
@@ -83,11 +83,12 @@ public function addTestExecutionInfo(iterable $traces): iterable
     private function testExecutionInfoAdder(iterable $traces): iterable
     {
         foreach ($traces as $trace) {
-            foreach ($trace->retrieveTestLocations()->byLine as &$testsLocations) {
+            foreach ($trace->retrieveTestLocations()->getTestsLocationsBySourceLine() as &$testsLocations) {
                 foreach ($testsLocations as $line => $test) {
-                    $testsLocations[$line] = self::createCompleteTestLocation($test);
+                    $testsLocations[$line] = $this->createCompleteTestLocation($test);
                 }
             }
+            unset($testsLocations);
```

Comment:
This loop with references is a trouble brewing, but I guess it's another time to fix.